Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -34,23 +40,47 @@ internal class RoutingTable
             }
 
             _address = address;
-            _tableSize = tableSize;
-            _logger = logger;
+            TableSize = tableSize;
+            BucketSize = bucketSize;
+            _logger = Log.ForContext<RoutingTable>();
 
-            _buckets = new KBucket[tableSize];
-            for (int i = 0; i < _tableSize; i++)
+            var random = new Random();
+            _buckets = new KBucket[TableSize];
+            for (int i = 0; i < TableSize; i++)
             {
-                _buckets[i] = new KBucket(bucketSize, random, _logger);
+                _buckets[i] = new KBucket(BucketSize, random, _logger);
             }
         }
 
+        /// <summary>
+        /// Number of buckets in the table.
+        /// </summary>
+        public int TableSize { get; }
+
+        /// <summary>
+        /// Size of single bucket.
+        /// </summary>
+        public int BucketSize { get; }
+
+        /// <summary>
+        /// Number of peers in the table.
+        /// </summary>
         public int Count => _buckets.Sum(bucket => bucket.Count);
 
+        /// <summary>
+        /// <see cref="IEnumerable{T}"/> of peers in the table.
```

Comment:
```suggestion
        /// List of peers in the table.
```

With the XML document, we will see a strange sentence like the below link.

![image](https://user-images.githubusercontent.com/26626194/101320027-56fbc800-38a6-11eb-913f-a5d3198accfc.png)


https://docs.libplanet.io/main/api/Libplanet.Net.ITransport.html#Libplanet_Net_ITransport_BootstrapAsync_IEnumerable_Libplanet_Net_BoundPeer__System_Nullable_TimeSpan__System_Nullable_TimeSpan__System_Int32_CancellationToken_

Reply:
@moreal That's a known issue (at least to me), and it's the problem with our build settings with [DocFX].  We shouldn't remove the `<see>` element due to this, but we should fix the DocFX configuration instead.

[DocFX]: https://dotnet.github.io/docfx/